extension/chrome/elements/composer/composer-pwd-or-pubkey-container.ts

```diff
@@ -53,7 +54,7 @@ export class ComposerPwdOrPubkeyContainer extends ComposerComponent {
       this.hideMsgPwdUi(); // Hide 'Add Pasword' prompt if there are no recipients or message is not encrypted
       this.composer.sendBtn.enableBtn();
     } else if (this.composer.recipients.getRecipients().find(r => r.status === RecipientStatuses.NO_PGP)) {
-      await this.showMsgPwdUiAndColorBtn();
+      this.showMsgPwdUiAndColorBtn(); // tslint:disable-line:no-floating-promises
```

Comment:
Instead of suppressing it, you can add `.catch(Catch.reportErr)` even if you don't expect any errors to be thrown.

Reply:
Done, thanks :+1: